app/models/taxon.rb

```diff
@@ -68,4 +74,11 @@ def can_subscribe?
 
     true
   end
+
+private
+
+  def fetch_tagged_content
+    taxon_content_ids = [content_id] + associated_taxons.map(&:content_id)
+    TaggedContent.fetch(*taxon_content_ids)
```

Comment:
Not sure I have a strong opinion on which is cleaner. I just didn't know what the splat did on this side of the message sending.